File: dynamic_listings/javascript/default/maths/probability_20.js
function main(){
    let happened = 0;
    // Monte Carlo Method we do 10000 "extractions"
    for (let i = 0; i < 1000; i++){
        // Get a random number between 1 and 5
        let n = Math.random() * 5 + 1;
        if (n == 1){
            // If it's 1, we have a match!
            happened++;
        }
    }
    // We print the result
    console.log(happened / 10000)
}

File: dynamic_listings/python/default/maths/probability_20.py
#!/usr/bin/env python3
import random


def main() -> None:
    happened: int = 0
    # Monte Carlo Method we do 10000 "extractions"
    for _ in range(10000):
        # Get a random number between 1 and 5
        n: int = random.randint(1, 5)
        if n == 1:
            # If it's 1, we have a match!
            happened += 1
    # We print the result
    print(happened / 10000)


if __name__ == '__main__':
    # This is some boilerplate to execute directly from shell
    main()
